### ISSUE TITLE
Does Istanbul support config files in .js format?

### ISSUE THREAD

**Issue by CharlieEarnup** [istanbuljs/nyc#789]
Does Istanbul support config files in .js format?

**Comment by bcoe** [comment]
@CharlieEarnup at this time we do not support `.js` format configuration. I've heard mixed opinions as to whether or not this is a good idea, the danger being that your configuration file develops more logic than is probably a great idea.

Do you have a specific use-case that requires this functionality?

**Comment by CharlieEarnup** [comment]
Nope. however it seems like a pretty standard feature nowadays, and JS is our format of choice for all config files. We haven't had any issue introducing logic into our eslintrc files 😄
